docs/sections/running_processes/workflows.md

```diff
@@ -141,32 +188,69 @@ Band structure computed by the `PwBandsWorkChain`.
 
 :::
 
-## Overriding the inputs
+:::{important}
+
+The protocols and the `get_builder_from_protocol()` have only been fairly recently implemented in `aiida-quantumespresso`!
+Be sure to check [the open issues on GitHub](https://github.com/aiidateam/aiida-quantumespresso/issues?q=is%3Aopen+is%3Aissue+label%3Atopic%2Fprotocol) related to this feature and run some tests before starting calculations in production.
+
+:::
```

Comment:
I would maybe put this at the end of the previous section no? It is strange that it is on "Displaying the results".

Reply:
Sure, will do!